entry_types/scrolled/lib/tasks/pageflow_scrolled_tasks.rake

```diff
@@ -72,6 +72,14 @@ namespace :pageflow_scrolled do
                   testReferenceName: 'interview_toni'
                 }
               }.stringify_keys
+            },
+            audio_files: {
+              rocky_theme: {
+                url: 'https://s3-eu-west-1.amazonaws.com/de.codevise.pageflow.development/pageflow-next/seed-assets/audios/Rocky_Theme.m4a',
+                configuration: {
+                  testReferenceName: 'rocky_theme'
+                }
+              }.stringify_keys
```

Comment:
Do we need to think about the license of this file?

Reply:
Not yet documented here: https://github.com/codevise/pageflow/blob/0c84a00949f2c34934ebd71da5306dcc5e0beb65/entry_types/scrolled/package/spec/support/stories.js#L72-L78